src/panels/lovelace/components/hui-theme-select-editor.ts

```diff
@@ -0,0 +1,71 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import "@polymer/paper-button/paper-button";
+import { TemplateResult } from "lit-html";
+
+import { HomeAssistant } from "../../../types";
+import { fireEvent } from "../../../common/dom/fire_event";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+
+export class HuiThemeSelectionEditor extends hassLocalizeLitMixin(LitElement) {
+  public value?: string;
+  protected hass?: HomeAssistant;
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      value: {},
+    };
+  }
+
+  protected render(): TemplateResult {
+    const themes = ["Backend-selected", "default"].concat(
+      Object.keys(this.hass!.themes.themes).sort()
+    );
+
+    return html`
+      ${this.renderStyle()}
+      <paper-dropdown-menu
+        label="Theme"
+        dynamic-align
+        @value-changed="${this._changed}"
+      >
+        <paper-listbox
+          slot="dropdown-content"
+          .sselected="${this.value || "Backend-selected"}"
+          .attrForSelected="theme"
+        >
+          ${
+            themes.map((theme) => {
+              return html`
+                <paper-item .theme="${theme}">${theme}</paper-item>
```

Comment:
Since you're using `attr-for-selected`, `theme` should be an attribute. So `theme="${theme}"`

Makes me wonder, did you test this?

Reply:
I saw that it was broken and meant to go back and fix it... I forgot last night. and tunnel vision the entity editor

Reply:
It works with theme being a property 🤷‍♂️ I was more meaning testing of the .attrForSelected and the .sselected that was breaking it.